ui/src/components/Timer.tsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from 'react';
+
+const Timer = () => {
```

Comment:
zamiast komponentu proponwałbym stworzenie hooka, potem będzie łatwiej to wysłać na backend w komponencie który będzie używał tego timera